crates/typst-library/src/math/op.rs

```diff
@@ -31,8 +31,7 @@ pub struct OpElem {
 impl LayoutMath for OpElem {
     #[tracing::instrument(skip(ctx))]
     fn layout_math(&self, ctx: &mut MathContext) -> SourceResult<()> {
-        let fragment =
-            ctx.layout_text(&TextElem::new(self.text().clone()).spanned(self.span()))?;
+        let fragment = ctx.layout_fragment(&self.text)?;
```

Comment:
```suggestion
        let fragment = ctx.layout_fragment(self.text())?;
```
it doesn't really make a difference here, but it's the way it's done everywhere else and I would like to keep things consistent and compatible for future changes